### PR TITLE
Update version for current images

### PR DESCRIPTION
**Pull request by ISO50**
Update to new node 10.x version

### CODE REVIEW

**Comment by ISO50** [comment]
👍

**Comment by ISO50** [comment]
Not sure if this is enough. The documentation is short ;)
If something is missing, please let me know.

**Comment by ISO50** [comment]
@kgalli What is the status of this PR and of this repository in general?

If support is needed please let me know.

**Comment by kgalli** [comment]
@ISO50 please update to latest node. I tend to update `current` and `lts` to the latest.

**Comment by kgalli** [comment]
@ISO50 please also have a look at the official repo. Seems there are changes we have to move over:
https://github.com/nodejs/docker-node/blob/3e179a85703a6688a26486729b4466a92e818a84/10/jessie/Dockerfile#L17

**Comment by ISO50** [comment]
@kgalli 
I've updated Current and LTS and added the the new GPG key.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .DS_Store
 .idea/
+.vscode
```
Comment (kgalli) [inline .gitignore:3]: Not a big fan of having editor/IDE configs in `.gitignore`. I prefer global setting (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). But due to the fact we already have `.idea` I can not complain :-) ... Just keep in mind that eventually those lines might be removed from `.gitignore`.

**Comment by kgalli** [comment]
@ISO50 sorry for the late response. Thanks for the update.

**Comment by kgalli** [comment]
👍

**Comment by ISO50** [comment]
👍

**Pull request merged**